### ISSUE TITLE
Split auto created sessions and manually saved sessions into different search-lenses

### ISSUE THREAD

**Issue by danielyrovas** [rmagatti/session-lens#30]

**Comment by rmagatti** [comment]
Hi @danielyrovas, could you elaborate on what you mean by this?

**Comment by danielyrovas** [comment]
My idea was that if you invoke the SaveSession command to save a session manually then you could access that in a seperate list with search lens than the the sessions that are saved on quit. I think if I use auto session for a while the number of sessions that I have saved will continually accrue and could make it more difficult than it needs to be to find a specific session. If there is 10 or so sessions that you want to have to come back to regularly, you could save them manually and then have them display in a separate list.

**Comment by rmagatti** [comment]
@danielyrovas I see. Maybe there's just a better way of achieving the same result. Could look into implementing frecency of some kind in the picker, so you'd get those 10 sessions you use the most ranked higher. Food for thought.

**Comment by danielyrovas** [comment]
yes that would likely be a cleaner way to provide the same functionality. A 'pin to top' method could also work.
Thanks for the awesome plugin, I'm liking it a lot atm.
